### PR TITLE
Add support for HEALTHCHECK directive

### PR DESCRIPTION
**Pull request by yiranwang52**
#54

### CODE REVIEW

File: lib/parser/dockerfile/healthcheck.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	if len(cmdIndices) < 2 {
+		return nil, base.err(fmt.Errorf("CMD not defined"))
+	}
+
+	flags, err := splitArgs(base.Args[:cmdIndices[0]])
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse interval")
+	}
+
+	var interval, timeout, startPeriod time.Duration
```
Comment (apourchet) [inline lib/parser/dockerfile/healthcheck.go]: Does docker not have defaults for these? Is it just 0?

Comment (yiranwang52) [reply lib/parser/dockerfile/healthcheck.go]: My understanding is docker daemon will handle 0. Need more test for this.

File: lib/parser/dockerfile/healthcheck.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	var retries int
+	for _, flag := range flags {
+		if val, ok, err := parseFlag(flag, "interval"); err != nil {
```
Comment (evelynl94) [inline lib/parser/dockerfile/healthcheck.go]: maybe these can be their own functions.

Comment (yiranwang52) [reply lib/parser/dockerfile/healthcheck.go]: I will handle that later.

File: lib/parser/dockerfile/healthcheck.go (hunk 1 above)
Comment (yiranwang52) [reply lib/parser/dockerfile/healthcheck.go]: Seems to work.

**Pull request merged**